.prowci/tide.yaml

```diff
@@ -38,7 +38,7 @@ items:
         serviceAccountName: tide
         containers:
         - name: tide
-          image: registry.svc.ci.openshift.org/ci/tide:latest
+          image: quay.io/kargakis/tide:workaround
```

Comment:
For future reference, we are working around https://github.com/kubernetes/kubernetes/issues/64594